<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Event_modify_dialog_base</class>
 <widget class="QDialog" name="Event_modify_dialog_base">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>349</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>历史事项修改</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="1,0,0,0,0,0,0,0,0">
   <item row="0" column="0">
    <widget class="QLabel" name="textLabel1">
     <property name="text">
      <string>分组序号：</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="m_pGroup_combobox"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="textLabel2">
     <property name="text">
      <string>事项分类：</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="m_pEvent_sort_combobox"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="textLabel3">
     <property name="text">
      <string>事项类型：</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="m_pEvent_type_combobox"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="textLabel4">
     <property name="text">
      <string>事项时间：</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QDateTimeEdit" name="m_pDatetime_edit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_pMs_edit">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="textLabel5">
     <property name="text">
      <string>事项组名：</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="m_pRtu_combobox"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="textLabel6">
     <property name="text">
      <string>对象名称：</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QComboBox" name="m_pObject_combobox"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="textLabel7">
     <property name="text">
      <string>事项描述：</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="m_pEvent_desc_edit"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="textLabel8">
     <property name="text">
      <string>值：</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="m_pValue_edit"/>
   </item>
   <item row="8" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_pModify_button">
       <property name="text">
        <string>修改</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>25</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_pDelete_button">
       <property name="text">
        <string>删除</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>26</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_pAdd_button">
       <property name="text">
        <string>增加</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>30</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>m_pGroup_combobox</tabstop>
  <tabstop>m_pEvent_sort_combobox</tabstop>
  <tabstop>m_pEvent_type_combobox</tabstop>
  <tabstop>m_pDatetime_edit</tabstop>
  <tabstop>m_pMs_edit</tabstop>
  <tabstop>m_pRtu_combobox</tabstop>
  <tabstop>m_pObject_combobox</tabstop>
  <tabstop>m_pEvent_desc_edit</tabstop>
  <tabstop>m_pValue_edit</tabstop>
  <tabstop>m_pModify_button</tabstop>
  <tabstop>m_pDelete_button</tabstop>
  <tabstop>m_pAdd_button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
